impress-org/givewp
# Multi-Form Goal query results are cached

Resolves #5365

## Description



The `MultiFormGoal/ProgressBar/Query` now only runs once per block instance per page load. This prevents running duplicate queries on page load while allowing for real-time results as donations are made.

I originally was looking into `Give_Cache` class. However, I changed direction for the following reasons:
1. Results are unique to each block instance.
1. Progress should be updated in real time as donations are made.

Therefor, the results are cached within the request cycle to avoid duplicate queries and each instance of the block model has its own cache (because the configuration will be different).

## Pre-review Checklist




## Testing Instructions



- Add a Multi-Form Goal Block to a page.
- Check that the Revenue table is only queried once
